src/Combobox/index.stories.js

```diff
@@ -253,13 +257,9 @@ export const WithActions = (args) => {
           label="Do Action"
         />
       </Combobox>
-      <Dialog
-        title="You selected an action"
-        isOpen={isOpen}
-        onUserDismiss={() => setIsOpen(false)}
-      >
+      <Drawer isOpen={isOpen} onUserDismiss={() => setIsOpen(false)}>
```

Comment:
`Drawer` is a better pick for the actions story because it doesn't steal focus by default.